tests/serializers/models/natural.py

```diff
@@ -91,3 +91,28 @@ class PostToOptOutSubclassUser(models.Model):
     subscribers = models.ManyToManyField(
         SubclassNaturalKeyOptOutUser, related_name="subscribed_posts", blank=True
     )
+
+
+class NaturalKeyWithNullableFieldManager(models.Manager):
+    def get_by_natural_key(self, name, optional_id):
+        return self.get(name=name, optional_id=optional_id)
+
+
+class NaturalKeyWithNullableField(models.Model):
+    name = models.CharField(max_length=100)
+    optional_id = models.UUIDField(null=True, blank=True)
```

Comment:
So my example of `UUIDField` on the ticket was just what I happened to be working with (and show a really nice validation problem when attempting the roundtrip). However, it's not very realistic as a member of a natural key. Could I ask you to try another field type?